server/app.ts

```diff
@@ -105,6 +106,8 @@ export default function createApp(
     setUpCreateBarcode(createBarcodeService, prisonRegisterService, contactService, recipientFormService)
   )
 
+  app.use('/', handleSlm404Errors())
```

Comment:
This is crucial that this is called after the SLM routes are all defined, but before the CR39 authentication and routes